### DISCUSSION TITLE
OpenBSD support? Or throw OpenBSD in the trash?

### DISCUSSION THREAD

**Discussion by cschech** [dankamongmen/notcurses#2718] [Ideas]
Probably too difficult because OpenBSD bundles an ancient ncurses and pretends it's just curses.h, plus LIBRT.

https://invisible-island.net/ncurses/ncurses-openbsd.html

**Comment by cschech** [comment]
thinkcentre# cmake build
-- Requested multimedia engine: ffmpeg
-- Requested build mode: RelWithDebInfo
-- Checking for one of the modules 'tinfo>=6.1;ncursesw>=6.1'
CMake Error at /usr/local/share/cmake/Modules/FindPkgConfig.cmake:894 (message):
  None of the required 'tinfo>=6.1;ncursesw>=6.1' found
Call Stack (most recent call first):
  CMakeLists.txt:104 (pkg_search_module)


CMake Error at CMakeLists.txt:153 (find_library):
  Could not find LIBRT using the following names: rt


-- Configuring incomplete, errors occurred!
See also "/root/notcurses/build/CMakeFiles/CMakeOutput.log".
See also "/root/notcurses/build/CMakeFiles/CMakeError.log".

**Comment by dankamongmen** [comment]
yeah https://invisible-island.net/ncurses/ncurses-openbsd.html does not fill me with confidence. i don't see myself spending time on this, as i know exactly zero people running openbsd.
